util/test/test_dftool.py

```diff
@@ -128,13 +128,16 @@ def test_noarg_commands():
         "dispense_active",
         "querymany",
         "compile",
-        "manyrandom",
-        "newdfrewards",
+        "many_random",
+        "new_dfrewards",
```

Comment:
Please move all changes wrt manyrandom, newdfrewards, newacct, newtoken, acctinfo, chaininfo to a new PR. (With an appropriate github issue attached.)

Why: each PR should be well-defined and tightly scoped. (As discussed above.)

Reply:
Moved to this PR => https://github.com/oceanprotocol/df-py/pull/506
Ticket => https://github.com/oceanprotocol/df-py/issues/505